testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -660,6 +679,53 @@ impl ClusterSwarmKube {
         )
         .await
     }
+
+    async fn config_fluentbit(&self, input_tag: &str, pod_name: &str, node: &str) -> Result<()> {
+        let parsers_config = include_str!("fluent-bit/parsers.conf").to_string();
+        let fluentbit_config = format!(
+            include_str!("fluent-bit/fluent-bit.conf"),
+            input_tag = input_tag,
+            pod_name = pod_name
+        );
+
+        let bucket = "toro-cluster-test-flamegraphs";
+        let run_id = thread_rng()
+            .sample_iter(&Alphanumeric)
+            .take(30)
+            .collect::<String>();
+        self.s3_client
+            .put_object(PutObjectRequest {
+                bucket: bucket.to_string(),
+                key: format!("data/{}/{}/fluent-bit/parsers.conf", run_id, pod_name),
+                body: Some(parsers_config.into_bytes().into()),
+                ..Default::default()
+            })
+            .await?;
+        self.s3_client
+            .put_object(PutObjectRequest {
+                bucket: bucket.to_string(),
+                key: format!("data/{}/{}/fluent-bit/fluent-bit.conf", run_id, pod_name),
+                body: Some(fluentbit_config.into_bytes().into()),
+                ..Default::default()
+            })
+            .await?;
+
+        let dir = "/opt/libra/data/fluent-bit/";
+        self.util_cmd(format!("mkdir -p {}", dir), node, "mkdir-fluentbit")
+            .await?;
+        self.util_cmd(
```

Comment:
1) For the `run_id`, can we use same value across all calls in the same run? I would propose this:
- In cti, there is pod_name(that sets name of cluster test pod, this is unique across al cluster test runs) and $WORKSPACE. You can pass env var(with -E), like RUN_ID to the cluster test container by combining this two, e.g. RUN_ID=$WORKSPACE-$pod_name.
- Then in cluster test code you can read run id with env::var

2) Since we are going to reuse this later for genesis, etc, do you mind adding method to KubeSwarm, like `put_file(path, content)`
And have this method doing put_object and then s3 cp. object name for s3 can be derived from host path(e.g. if we put file to `/opt/libra/data/fluent-bit/xxx`, you can derive s3 path as `data/{run}/{pod}/opt/libra/data/fluent-bit/xxx`)